### ISSUE TITLE
[Feature]: Support for Parcel

### ISSUE THREAD

**Issue by ivinoop** [wallabyjs/console-ninja#112]
### Describe the feature you'd like.

Please extend/enhance support for Parcel based projects. Currently supported bundlers are Vite, Webpack, etc.

[Parcel ](https://parceljs.org/)

**Comment by Contrapuntist** [comment]
yeah, I was trying to figure out why the extension wasn't working with a parcel project. It wasn't immediate clear that Parcel isn't supported.

**Comment by finkusuma-dev** [comment]
Please, I also need the support for Parcel. Is there any reason why Parcel is not being supported?

**Comment by smcenlly** [comment]
Console Ninja integrates with your development tools to process your files before they are compiled within the JavaScript runtime environment (e.g. node environment).

In contrast, Parcel's JavaScript compiler is written in Rust for native performance and executes outside of the node environment. Right now, we don't have a mechanism to integrate current Console Ninja functionality within Rust.
